Operators/3/Date.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Operators
+{
+    internal class Date
+    {
+        public int year, month, day;
+
+        public Date(int year, int month, int day)
+        {
+            this.year = year;
+            this.month = month;
+            this.day = day;
+        }
+
+        public static Date operator -(Date d1, Date d2)
+        {
+            return new Date(d1.year - d2.year, d1.month - d2.month, d1.day - d2.day);
+        }
+
+        public static Date operator +(Date d1, int days)
```

Comment:
What this method does ?

Reply:
It was in the assignment "Using the operator overload mechanism, define the operation of increasing the date by a certain number of days." And I used overload operator "+" .